packages/scandipwa/src/component/FieldFile/FieldFile.container.js

```diff
@@ -119,9 +119,9 @@ export class FieldFileContainer extends PureComponent {
             setRef,
             events: {
                 ...events,
-                onChange: this.onChange.bind(this),
-                onClick: resetFieldValue.bind(this, this)
+                onChange: this.onChange.bind(this)
             },
+            resetFieldValue: resetFieldValue.bind(this, this),
```

Comment:
Please check the reason why `this` is passed twice. I believe this is a bug.

Reply:
@AzizKHAN030 Did you check this? What was the reason to pass this twice?

Reply:
I passed the field context as a parameter in order to set the state of the field from the parent component. Now I refactored it and I'm passing an object that has a method which calls setState.